srcpkgs/linux-asahi/template

```diff
@@ -0,0 +1,325 @@
+# Template file for 'linux-asahi'
+pkgname=linux-asahi
+_asahiver=6.10.6
+_asahirel=1
+_tag=asahi-${_asahiver}-${_asahirel}
+version=6.10.6
+revision=1
+_upstreamver=6.10.6
+_kernver="${_upstreamver}-${_tag}_${revision}"
```

Comment:
`$version` should probably be `$_tag`.

Overall, the entire version handling is a bit messy here, and could be cleanup up my using bash's brace expansion

Reply:
Cannot set version to _tag, accordint to the linter.
Messy but working :)

Reply:
Moreover, the version is not allowed to contain `-` or `_` symbols.
Let me know if you find a more clever way to deal with this.